Add email based auth with PKCE flow (#16683)
supabase/supabase: apps/docs/pages/guides/auth/auth-helpers.mdx

export const meta = {
  id: 'index',
  title: 'Server-Side Auth Overview',
  description: 'Server-Side Auth guides and utilities for working with Supabase.',
  sidebar_label: 'Overview',
}

> Working with server-side frameworks is a bit different to client-side frameworks, in this section we cover the various ways of handling authentication on the server-side and provide some helper libraries to make the process more seamless when working with Supabase.

Reviewer: ```suggestion
Working with server-side frameworks is slightly different to client-side frameworks. In this section we cover the various ways of handling server-side authentication and demonstrate how to use the Supabase helper-libraries to make the process more seamless.
```
